docs/index.md

```diff
@@ -339,8 +340,15 @@ val printThings = FunSpec.builder("printThings")
   .addParameter("things", producerArrayOfThings)
   .addStatement("println(things)")
   .build()
+
+val printKClass = FunSpec.builder("printKClass")
+  .addParameter("kClass", KClass::class.asClassName().parameterizedBy(STAR))
+  .addStatement("println(kClass)")
+  .build()
 ```
 
+The `STAR` is represented as `*` in KotlinPoet. You can find more in [`com.squareup.kotlinpoet.TypeName`](https://github.com/square/kotlinpoet/blob/master/kotlinpoet/src/main/java/com/squareup/kotlinpoet/TypeName.kt)
```

Comment:
I think it's better to link to KDoc rather than sources. Plus, linking to an entire file might not be that helpful.